google/cloud/storage/bucket_test.cc

```diff
@@ -309,19 +310,53 @@ TEST_F(BucketTest, GetBucketIamPolicy) {
 TEST_F(BucketTest, GetBucketIamPolicyTooManyFailures) {
   testing::TooManyFailuresTest<IamPolicy>(
       mock, EXPECT_CALL(*mock, GetBucketIamPolicy(_)),
-      [](Client& client) {
-        client.GetBucketIamPolicy("test-bucket-name");
-      },
+      [](Client& client) { client.GetBucketIamPolicy("test-bucket-name"); },
       "GetBucketIamPolicy");
 }
 
 TEST_F(BucketTest, GetBucketIamPolicyPermanentFailure) {
   testing::PermanentFailureTest<IamPolicy>(
       *client, EXPECT_CALL(*mock, GetBucketIamPolicy(_)),
+      [](Client& client) { client.GetBucketIamPolicy("test-bucket-name"); },
+      "GetBucketIamPolicy");
+}
+
+TEST_F(BucketTest, SetBucketIamPolicy) {
+  IamBindings bindings;
+  bindings.AddMember("storage.buckets.list", "test-user");
```

Comment:
Same nit as before about using permission names vs role names in examples/tests - call this storage.objectViewer, or some actual role, instead?